lib/sycamore/sycamore/utils/time_trace.py

```diff
@@ -95,3 +135,45 @@ def wrapper(*args, **kwargs):
         return wrapper
 
     return decorator
+
+
+class _ZeroRU:
+    def __init__(self):
+        self.ru_utime = 0
+        self.ru_stime = 0
+
+
+class LogTime:
+    def __init__(self, name, point: bool = False, log_start: bool = False):
```

Comment:
Might want to add `, /, *,` after `name` to force keyword usage.

Reply:
done, although the internet tells me it's just `, *,`